letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -488,13 +488,56 @@ BootstrapRpmCommon() {
   BootstrapRpmCommonBase "$python_pkgs"
 }
 
+# If new packages are installed by BootstrapRpmPython3 below, this version
+# number must be increased.
+BOOTSTRAP_RPM_PYTHON3_LEGACY_VERSION=1
+
+BootstrapRpmPython3Legacy() {
+  # Tested with:
+  #   - CentOS 6
+
+  InitializeRPMCommonBase
+
+  if ! $TOOL list rh-python36 >/dev/null 2>&1; then
+    echo "To use Certbot, packages from the SCL repository need to be installed."
```

Comment:
I would say (here and in the other case) "To use Certbot ****on this operating system****", because some user might think that this is an absolute requirement for using Certbot in general.

Reply:
Added. For the sake of consistency, I also updated the similar message in `rpm_common_base.sh`.